sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -910,4 +910,45 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-35686: error out for creating view with auto gen alias") {
```

Comment:
shall we put the tests in `PersistedViewTestSuite`?

Reply:
currently the SQLViewTestSuite only supports run a case against all 3 types. it's not able to specify a single view type. Maybe we can start to unify the `SQLViewSuite` and `SQLViewTestSuite`?